packages/block-library/src/column/index.js

```diff
@@ -24,17 +24,7 @@ export const settings = {
 		inserter: false,
 		reusable: false,
 		html: false,
-	},
-	getEditWrapperProps( attributes ) {
-		const { width } = attributes;
-		if ( Number.isFinite( width ) ) {
-			return {
-				style: {
-					flexBasis: width + '%',
-				},
-				'data-has-explicit-width': true,
-			};
-		}
+		lightBlockWrapper: true,
```

Comment:
Is this experimental? Where do we document it?

Reply:
Ah, it should have been prefixed as well with `__experimental`.